aikido_perception/include/aikido/perception/AprilTagsModule.hpp

```diff
@@ -0,0 +1,68 @@
+/** 
+ * @file AprilTagsModule.hpp
+ * @author Shushman Choudhury
+ * @date Apr 14, 2016
+ * @brief The header for the April Tags detector. The function detectObjects
+ * waits for the next ROS message on the specified topic.
+ */
+
+#ifndef AIKIDO_PERCEPTION_APRILTAGSMODULE_H
+#define AIKIDO_PERCEPTION_APRILTAGSMODULE_H
+
+#include <string>
+#include <ros/ros.h>
+#include <ros/forwards.h>
+#include <ros/single_subscriber_publisher.h>
+#include <aikido/perception/ConfigDataLoader.hpp>
+#include <tf/transform_listener.h>
+#include "yaml-cpp/yaml.h"
+
+#include <dart/dart.h>
+#include <aikido/util/CatkinResourceRetriever.hpp>
+
+#include "PerceptionModule.hpp"
+
+
+namespace aikido{
+namespace perception{
+
+/// The April Tag detector 
+class AprilTagsModule : public virtual PerceptionModule
```

Comment:
Add Doxygen docstrings to all functions in this library. For functions, they should also include arguments.